lib/network.go

```diff
@@ -1914,6 +1915,14 @@ type MsgDeSoHeader struct {
 	// event that ASICs become powerful enough to have birthday problems in the future.
 	ExtraNonce uint64
 
+	// TransactionsConnectStatus is only used for Proof of Stake blocks, starting with
+	// MsgDeSoHeader version 2. For all earlier versions, this field will default to nil.
+	//
+	// The hash of the TxnConnectStatusByIndex field in MsgDeSoBlock. It is stored to ensure
+	// that the TxnConnectStatusByIndex is part of the header hash, which is signed by the
+	// proposer. The full index is stored in the block to offload space complexity.
+	TxnConnectStatusByIndexHash *BlockHash
```

Comment:
I thought we said we'd add it to the existing merkle root rather than adding another field to the header for this. What made us change?